framework/console/Controller.php

```diff
@@ -102,7 +102,7 @@ public function runAction($id, $params = [])
                 if (in_array($name, $options, true)) {
                     $default = $this->$name;
                     if (is_array($default)) {
-                        $this->$name = preg_split('/(?!\(\d+)\s*,\s*(?!\d+\))/', $value);
+                        $this->$name = preg_split('/(?!\((\' | ")*[^"\']+)\s*,\s*((?![^"\'(\)]*(\'|")*\)))/', $value);
```

Comment:
again i am no good at regex, i don't know what I am seeing or how does it affect the commands.

In this case

What will do to

`yii fixture *, -User`

how about

`yii fixture -User -n="@common/fixtures"`